Actions/CreateApp/AppHelper.psm1

```diff
@@ -212,7 +212,13 @@ function Add-NewAppFolderToWorkspaceFolders
 )
 {
     $newAppFolder = [PSCustomObject]@{ "path" = $appFolder }
-    $afterFolder = $workspaceFolders | Where-Object { $_.path -ne '.github' -and $_.path -ne '.AL-Go' } | Select-Object -Last 1
+
+    if ($workspaceFolders){
+        $afterFolder = $workspaceFolders | Where-Object { $_.path -ne '.github' -and $_.path -ne '.AL-Go' } | Select-Object -Last 1
```

Comment:
Good catch to ensure that there are workspace folders, but this construct is hard to read.

Please do:

```
if (-not $workspaceFolders) {
    return @($newAppFolder)
}
```

Please also add a test for this scenario.
(Like the last one - just with null instead of an empty array - result should be the same I guess)

Reply:
True, I got a bit too roundabout there.